mycroft/configuration/config.py

```diff
@@ -128,17 +132,20 @@ class ConfigurationManager(object):
     _config = None
 
     @staticmethod
-    def load(*config_files):
+    def load(defaults=CONFIG_LOCATIONS, *config_files):
```

Comment:
Yeah, it definitely didn't do what I intended it to do. It might have worked in python3 but I'm not 100% sure of that either.